tests/tests/test_end_to_end_api_test.py

```diff
@@ -0,0 +1,176 @@
+                        negative=f"Failed to retrieved the rating. Response: {assessment_response.json() if assessment_response else assessment_response}."
+                    )
+
+                    assessment_result_flag = assessment_response and assessment_response.status_code == 200
+                    assessment_id = assessment_response.json().get('id') if assessment_result_flag else None
+
+                    test_api_obj.log_result(
+                        assessment_result_flag,
+                        positive=f"Successfully retrieved assessment ID: {assessment_id}",
+                        negative=f"Failed to retrieve assessment ID. Response: {assessment_response.json() if assessment_response else assessment_response}."
+                    )
+
+                    if assessment_result_flag:
+                        rating_payload = {
+                            'capability_assessment_id': assessment_id,
+                            'rating': rating_details,
+                            'comment': 'Retrieving rating details'
+                        }
+
+                        rating_response = test_api_obj.submit_ratings(assessment_id=assessment_id, rating_details=rating_payload, auth_details=auth_details)
+
+                        rating_result_flag = rating_response and rating_response.status_code == 200
+                        test_api_obj.log_result(
+                            rating_result_flag,
+                            positive=f"Successfully submitted rating with details: {rating_response.json()}",
+                            negative=f"Failed to submit rating with response: {rating_response.json() if rating_response else rating_response}."
+                        )
+
+            # Delete ACC model
+            if acc_model_result_flag:
+                delete_response = test_api_obj.delete_acc_model(acc_model_id=acc_model_id, auth_details=auth_details)
+                delete_result_flag = delete_response and delete_response.status_code == 200
+
+                test_api_obj.log_result(
+                    delete_result_flag,
+                    positive=f"Successfully deleted ACC model with ID: {acc_model_id}",
+                    negative=f"Failed to delete ACC model. Response: {delete_response.json() if delete_response else delete_response}."
+                )
+
+            # Delete attributes
+            if attribute_result_flag:
+                delete_response = test_api_obj.delete_attribute(attribute_id=attribute_id, auth_details=auth_details)
+                delete_result_flag = delete_response and delete_response.status_code == 200
+
+                test_api_obj.log_result(
+                    delete_result_flag,
+                    positive=f"Successfully deleted attribute with ID: {attribute_id}",
+                    negative=f"Failed to delete attribute. Response: {delete_response.json() if delete_response else delete_response}."
+                )
+

```

Comment:
Would be good to check for 403 and 401 errors. Example as done in the test_api_example.py